telegram_service.py

```
@@ -0,0 +1,276 @@
+        def _send():
+            try:
+                payload = {
+                    "task_id": getattr(task, "task_id", ""),
+                    "kind": getattr(task, "kind", ""),
+                    "status": getattr(task, "status", ""),
+                    "error": getattr(task, "error", None),
+                    "result": str(getattr(task, "result", ""))[:2000],
+                }
+                with httpx.Client(timeout=10.0) as client:
+                    client.post(self.webhook_url, json=payload)
+            except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)